src/sgame/sg_buildable.cpp

```diff
@@ -1702,6 +1702,40 @@ itemBuildError_t G_CanBuild( gentity_t *ent, buildable_t buildable, int /*distan
 		level.numBuildablesForRemoval = 0;
 	}
 
+	int max_miners = g_maxMiners.Get();
+	if (max_miners >= 0)
```

Comment:
Since yuo did the `< 0` => disabled, this needs to be `max_miners > 0` and if `==0` always return `IBE_NOHUMANBP : IBE_NOALIENBP`

Reply:
That would yield the same results, and only optimize the case max_miners=0. I think it's fine without that optimization right now.

Reply:
I agree with Sweet here